tests/drivers/spi/spi_loopback/boards/nucleo_h743zi.conf

```diff
@@ -1,7 +1,6 @@
 CONFIG_SPI_LOOPBACK_DRV_NAME="SPI_1"
 
-CONFIG_SPI_STM32_DMA=y
-CONFIG_SPI_STM32_INTERRUPT=n
+CONFIG_SPI_STM32_INTERRUPT=y
 CONFIG_SPI_ASYNC=n
```

Comment:
Should you remove `CONFIG_SPI_ASYNC=n` like for other boards ?

Reply:
yes, removed